packages/v4/src/content/accessibility/skip-to-content/skip-to-content.md

```diff
@@ -0,0 +1,16 @@
+---
+id: Skip to content
+section: components
+---
+
+A **skip to content** is a component that allows screen reader and keyboard users to bypass navigation rather than tabbing through it.
+
+**Keyboard users** should be able to find the skip to content using **Tab** at the start of a page with navigation. They should be able to select the skip to content using **Enter**.
+
+**Screen reader users** should also be able to navigate to skip to content using **Tab** and select it to skip past the navigation of a page.
+
+A skip to content does not trap tab focus. It is the first tabbable element on a page, but can be tabbed past and into the navigation of a page if the user prefers.
+
+## To make skip to content accessible:
+- Make sure the skip to content can be accessed with **Tab** 
```

Comment:
I don't know if I'd add this in the recommendation section because technically every component should be tested. I think this section should be more about when they're creating it.